.travis.yml

```diff
@@ -0,0 +1,39 @@
+language: go
+
+go:
+  - 1.3
+  - 1.4
+  - tip
+
+before_install:
+  # install linting tools
+  - go get github.com/golang/lint/golint
+  - go get golang.org/x/tools/cmd/vet
+  - go get github.com/fzipp/gocyclo
+  # install code coverage tools
+  - go get github.com/axw/gocov/gocov
+  - go get github.com/mattn/goveralls
+  - if ! go get code.google.com/p/go.tools/cmd/cover; then go get golang.org/x/tools/cmd/cover; fi
+
+install:
+  - cd lib/goose
+  - go get -t
+  - go build
+  - cd ../..
```

Comment:
using `cd` is more likely to introduce breakage.
For example, on line 30, you `cd cmd/goose`. Everything from then on is run in that directory. This build script skips `lib/goose` entirely. I could break something in `lib/goose` and it'd never be detected.